nuklear.h

```diff
@@ -13904,6 +13921,10 @@ nk_input_end(struct nk_context *ctx)
         in->mouse.ungrab = 0;
         in->mouse.grab = 0;
     }
+    for (int i = 0; i < NK_BUTTON_MAX; ++i) {
```

Comment:
This line is invalid C89 code. C89 does now allow the definition of variables in the for statement. I recommend copying the style from the ``` nk_input_begin ``` function to fix this issue.